packages/tracing/src/browser/metrics.ts

```diff
@@ -224,14 +232,26 @@ function addResourceSpans(
     return undefined;
   }
 
+  const data: Record<string, any> = {};
+  if (entry.transferSize) {
```

Comment:
`transferSize` can be 0 if it's pulling from local, which won't pass this condition 😄

Reply:
Use `'transferSize' in entry` everywhere instead :)

Reply:
Yup, thats my bad, will fix.